File: Arreglos o Vectores/Ejercicio5-Arrays.cpp
/*5. Desarrolle un programa que lea de la entrada estándar un vector de enteros y determine el mayor elemento del vector.*/

#include<iostream>
#include<stdlib.h>

using namespace std;

int main (){
    int numeros[100], n,  mayor=0;
    cout<<"Digite numero de elementos: ";
    cin>>n;
    for (int i=0; i<n; i++){
        cout<<i+1<<". Digite un numero: "; // 1.Digite un numero: 
        cin>>numeros[i];

        if (numeros[i] > mayor){
            mayor = numeros[i];
        }
    }
    cout<<"\nEl numero mayor en el vector es: "<<mayor<<endl;

    system ("pause");
    return 0;
}

File: Comparativa Python/Ejercicio5-Arrays.py
"""5. Desarrolle un programa que lea de la entrada estándar un vector de
enteros y determine el mayor elemento del vector."""

miLista = []

n = int(input('Digite el número de elementos que tendrá el arreglo: '))
mayor = 0
for i in range(n):
    miLista.append(int(input('{}. Digite un número: '.format(i+1))))

    if mayor < miLista[i]:
        mayor = miLista[i]

print('\nEl elemento más grande en el array es: ', mayor)
